src/crypto.ts

```diff
@@ -47,6 +47,25 @@ function encodeNumber(num: number): Uint8Array {
   return encoded;
 }
 
+/**
+ * Converts the given bigint into a uint8 array.
+ *
+ * @param int - Bigint to encode.
+ * @returns - Bigint encoded as a byte array.
```

Comment:
Can we add `@throws` that explains that it might throw if the int is too big? I wouldn't expect that, so it would be helpful.